dbf2csv/wizard/dbf2csv.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 INGETIVE (<https://ingetive.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import tempfile
+import csv
+import base64
+import logging
+
+from odoo import models, fields, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from dbfpy import dbf
+except ImportError as err:
+    _logger.info(err)
+
+
+class Conversion(models.TransientModel):
+    _name = 'dbftocsv.conversion'
+    _description = 'File DBF to convert to CSV'
+
+    data_file = fields.Binary(
+        string='DBF file',
+        required=True,
+        help='Get you DBF file.')
+    filename = fields.Char(string='DBF file')
+
+    data_file_csv = fields.Binary(
+        string='CSV file',
+        readonly=True,
+        help='Get you CSV file.')
+    filename_csv = fields.Char(string='CSV file')
+
+    def reopen_wizard(self):
+        return {
+            'type': 'ir.action.act_windows',
+            'name': 'Conversion',
+            'res_model': 'dbftocsv.conversion',
+            'res_id': self.id,
+            'view_type': 'form',
+            'view_mode': 'form',
+            'target': 'new'}
```

Comment:
[OCA guideline](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms): When making a comma separated list, dict, tuple, ... with one element per line, append a comma to the last element. This makes it so the next element added only changes one line in the changeset instead of changing the last element to simply add a comma.

Reply:
Ok, :memo: